scalopus_tracing/src/lttng/lttng_source.cpp

```diff
@@ -131,10 +137,41 @@ std::vector<json> LttngSource::convertEvents()
         entry["ph"] = "i";
         entry["s"] = "t";
       }
+      else if (event.name() == "counter_event")
```

Comment:
It seems that in the rest of the code you call them `count events` but here it's called `counter_event`. Maybe change the event name :thinking:

Reply:
The event name is actually already a `count_event`, just forgot to change the comparison here. Good catch.